config.m4

```diff
@@ -1,21 +1,10 @@
 dnl $Id$
 dnl config.m4 for extension phpy
 
-PHP_ARG_WITH([python_dir],
-  [dir of python],
-  [AS_HELP_STRING([[--with-python-dir[=DIR]]],
-    [Specify python installation dir, default is /opt/anaconda3])], [no], [no])
-
-
 PHP_ARG_WITH([python_version],
   [version of python],
   [AS_HELP_STRING([[--with-python-version[=VERSION]]],
-    [Specify version of python, default is 3.11])], [no], [no])
-
-PHP_ARG_WITH([python_config],
-  [path of python_config],
```

Comment:
It is necessary to retain the `python-config` option. When using Anaconda to manage Python, and multiple versions of Python are present, it is essential to utilize absolute paths.

Reply:
I don't even know what Anaconda is (seems very distro specific)

BTW, `python-config` and `pkg-config` will give you same results. `pkg-config` is used by PHP since 7.4 for most of the libraries. For non standard installation, setting PKG_CONFIG_PATH is the proper way.

Feel free to keep this PR as a sample of clean config.m4 and adapt to your needs